packages/common/src/components/UvText/UvText.tsx

```diff
@@ -265,7 +265,7 @@ const UvText: React.FC<Props> = ({ question }) => {
         if (textValueDate.date != null && textValueDate.date.length > 0) {
           displayText = textValueDate.date
         } else {
-          displayText = ''
+          displayText = 'Ej angivet'
```

Comment:
If uncertain date is selected for example, will this display "Ej angivet" for dodsdatum?

Reply:
No, uncertain date is handled in the next case statement (lines 272-279). Maybe this needs to be updated too?

Reply:
I dont think we need to set displayText if no value is present, since it is set to "Ej angivet" by default in line 188